### PR TITLE
Add unplug airbeam view to sd sync flow

### PR DESCRIPTION
**Pull request by anna1901**
https://trello.com/c/M99IsAKP

### CODE REVIEW

File: AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABView.swift [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    
+    var messageLabel: some View {
+        Text(Strings.UnplugAirbeamView.message)
+            .font(Fonts.regularHeading1)
+            .foregroundColor(.aircastingGray)
+    }
+    
+    var continueButton: some View {
+        Button {
+            viewModel.presentNextScreen = true
```
Comment (jankrzempek) [inline AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABView.swift:55]: As I remember we have used in others SD sync views func in ViewModel to do it, and then just call that function here. Please do it here too 😄

**Review by jankrzempek — COMMENTED** [review]
Great, awesome, sweet work!

File: AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABView.swift (hunk 1 above)
Comment (pawelgil) [reply AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABView.swift:55]: +1

File: AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABView.swift [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        NavigationLink(
+            destination: SDRestartABView(viewModel: SDRestartABViewModelDefault(urlProvider: viewModel.urlProvider), creatingSessionFlowContinues: $creatingSessionFlowContinues),
+            isActive: $viewModel.presentNextScreen,
+            label: {
+                EmptyView()
+            })
+    }
+}
+
+struct UnplugAirBeamView_Previews: PreviewProvider {
```
Comment (pawelgil) [inline AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABView.swift:71]: #if DEBUG

File: AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Created by Lunar on 08/12/2021.
+//
+
+import Foundation
+
+protocol UnplugABViewModel: ObservableObject {
+    var presentNextScreen: Bool { get set }
```
Comment (pawelgil) [inline AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABViewModel.swift:7]: after making changes Janek asked for this should become get-only (an issue with navigation link will arise, but you can refer to other PRs on how to handle it)

**Review by jankrzempek — APPROVED** [review]
Bellissimo 🤌🏻

**Pull request merged**